### PR TITLE
Updated FSEC and EAT Lancet 2.0 start scripts

### PR DESCRIPTION
**Pull request by FelicitasBeier**
## :bird: Description of this PR :bird:

- updated run script for EAT 2p0
- changed FSEC config for revision: 1. added labor8p5; labor1p9 for sensitivity runs (also change in start script);  moved PAL to institutions such that it is included in HumanWelfare scenario; deactivated yield calib for FSEC (as current default); updated input data (calibration file)

## :wrench: Checklist for PR creator :wrench:

- [x] Label pull request [from the label list](https://github.com/magpiemodel/magpie/labels).
  - **Low risk**: Simple bugfixes (missing files, updated documentation, typos) or changes  in start or output scripts
  - **Medium risk**: Uncritical changes in the model core (e.g. moderate modifications in non-default realizations)
  - **High risk**: Critical changes in model core or default settings (e.g. changing a model default or adjusting a core mechanic in the model)

- [x] Self-review own code
  - No hard coded numbers and cluster/country/region names.
  - The new code doesn't contain declared but unused parameters or variables.
  - [`magpie4`](https://github.com/pik-piam/magpie4) R library has been updated accordingly and backwards compatible where necessary.
  - `scenario_config.csv` has been updated accordingly (important if `default.cfg` has been updated)

- [x] Document changes 
  not necessary. only updated start script

- [x] Perform test runs
  - runs are made for EAT 2p0 submission

### :chart_with_downwards_trend: Performance changes :chart_with_upwards_trend:
  
  - Current develop branch default : ** mins
  - This PR's default :  ** mins

## :rotating_light: Checklist for reviewer :rotating_light:

- PR is labeled correctly
- Code changes look reasonable
  - No hard coded numbers and cluster/country/region names.
  - No unnecessary increase in module interfaces
  - model behavior/performance is satisfactory.
- Changes are properly documented
  - `CHANGELOG` is updated correctly
  - Updates in interfaces have been properly addressed in the module documentations
  - In-code documentation looks appropriate
- [x] content review done (at least 1)
- [x] RSE review done (at least 1)

### CODE REVIEW

**Review by pfuehrlich-pik — APPROVED** [review]

File: config/scenario_fsec.csv [hunk 1]
```diff
@@ -1,75 +1,77 @@
...
+gms$s15_exo_roots;;;;;;;;0;0;1;0;0;0;0;0;0;;;;;;;;;;;;;;;;;1;1;;;;;;;;;;;;;;;;;
+gms$s15_exo_pulses;;;;;;;;0;0;0;0;0;1;0;0;0;;;;;;;;;;;;;;;;;1;1;;;;;;;;;;;;;;;;;
+gms$s15_exo_sugar;;;;;;;;0;0;0;0;0;0;1;0;0;;;;;;;;;;;;;;;;;1;1;;;;;;;;;;;;;;;;;
+gms$s15_exo_oils;;;;;;;;0;0;0;0;0;0;1;0;0;;;;;;;;;;;;;;;;;1;1;;;;;;;;;;;;;;;;;
+gms$s15_exo_scp;;;;;;;;0;0;0;0;0;0;1;0;0;;;;;;;;;;;;;;;;;1;1;;;;;;;;;;;;;;;;;
+gms$s15_exo_alcohol;;;;;;;;0;0;0;0;0;0;1;0;0;;;;;;;;;;;;;;;;;1;1;;;;;;;;;;;;;;;;;
+gms$s15_alc_scen;0.014;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+gms$c15_food_scenario;;;SSP1;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+gms$c21_trade_liberalization;;;;;;;;;;;;;;;;;;;;;;;l908080r807070;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+gms$c22_protect_scenario;;;;;;;;;;;;;;;;;;;;;;;;;;;;;BH_IFL;;;;;;;;;;;;;;;;;;;;;;
```
Comment (pvjeetze) [inline config/scenario_fsec.csv:26]: I would suggest switching this to 30by30

Comment (pvjeetze) [inline config/scenario_fsec.csv:26]:
```suggestion
gms$c22_protect_scenario;;;;;;;;;;;;;;;;;;;;;;;;;;;;;30by30;;;;;;;;;;;;;;;;;;;;;;
```

Comment (pvjeetze) [reply config/scenario_fsec.csv:26]: @bodirsky?

**Review by pvjeetze — CHANGES_REQUESTED** [review]
Looks good to me. I would, however, as suggested earlier change the conservation template from `BH_IFL` to `30by30`

**Review by bodirsky — APPROVED** [review]

File: config/scenario_fsec.csv (hunk 1 above)
Comment (FelicitasBeier) [reply config/scenario_fsec.csv]: alright. I changed it now. Thanks!

**Review by pvjeetze — APPROVED** [review]

**Comment by pvjeetze** [comment]
Thanks, looks good to me!

**Review by FelicitasBeier — COMMENTED** [review]
PLEASE DO NOT MERGE, YET.
We want to include new input data.

**Pull request merged**